05-Introducing-Rigid-Body-Molecular-Dynamics/01-Introduction-to-Rigid-Bodies.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<ul><li>All list entires need periods.</li><li>Remove "Under the assumption of a uniform density distribution"</li></ul><p> </p><p>Let's talk about the wording on some of these definitions. I don't want to confuse new users.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>